packages/pages/src/components/address/methods.ts

```diff
@@ -98,7 +98,7 @@ export const getDirections = (
         listings?.reduce(
           (obj, listing) => ({
             ...obj,
-            [listing.publisher.toUpperCase()]: listing.listingUrl,
+            [(listing.publisher || "").toUpperCase()]: listing.listingUrl,
```

Comment:
It feels strange to have an `object` that can have `''` as a key. How do we handle if multiple listings are without a `publisher`? The same entry in the `object` will get clobbered. 

Actually, do we even care about the case where there is no `publisher`? It seems like below we're only looking for one key in the map: `ListingPublisherOption.GOOGLEMYBUSINESS`. Could we get rid of the `listingsMap` concept entirely and just do a `filter`?